lms/views/lti_launches.py

```diff
@@ -101,7 +107,12 @@ def handle_lti_launch(request, token=None, lti_params=None, user=None, jwt=None)
        as a canvas file
     """
     lti_key = lti_params['oauth_consumer_key']
-    context_id = lti_params['context_id']
+
+    try:
+        context_id = lti_params['context_id']
+    except KeyError:
+        raise MissingLTILaunchParamError('Context Id is required for lti launch.')
```

Comment:
Same note here as above: let's be more precise in this error string, e.g. `"LTI data parameter context_id is required for launch"`